### DISCUSSION TITLE
Is it possible to generate a VS solution for CoreCLR?

### DISCUSSION THREAD

**Discussion by FireBanana** [dotnet/runtime#80013] [Q&A] [answered]
I have the repo cloned but I'd like to generate project files for VS to navigate through the codebase, is this possible?

**Comment by En3Tho** [comment, marked as answer]
https://github.com/dotnet/runtime/blob/main/docs/workflow/debugging/coreclr/debugging-runtime.md

**Reply by FireBanana** [reply]
Thanks, I missed that document 🙂

**Comment by En3Tho** [comment]
Note that generated solution can also be used with JetBrains Rider on Windows.

**Reply by berkansasmaz** [reply]
Hello, @En3Tho. Could you perhaps elaborate on your JetBrains Rider experiences? My operating system is macOS, and I use rider, but I'm having trouble debugging the tests. Please specify what you did for JetBrains Rider in addition to the related contribution documents so that I understand what I'm missing.

**Reply by En3Tho** [reply]
@berkansasmaz Sorry, can't help you with MacOS. Only tested this on Windows (As I stated above). On Windows there is a solution that you can open both with Visual Studio and Rider. I tried this stuff on Linux some time ago but couldn't really do much. I guess CLion is the way on Linux and possibly MacOS too.

**Reply by berkansasmaz** [reply]
Thank you for your prompt response. Surprisingly, I can run tests against some libraries but not others. For instance, I can run tests in `System.Text.Json`, but I am unable to run tests in `System.Memory`. If I find a solution, I'll put it here; once again, thank you for your interest.
